### DISCUSSION TITLE
Transfer Request for linguify to typst-community

### DISCUSSION THREAD

**Discussion by jomaway** [typst-community/org#12] [Transfer Requests]
### Project Name

linguify

### Initial Author(s)

jomaway

### Repository URL

https://github.com/jomaway/typst-linguify

### Status

Unmaintained

### Description

Linguify is a package to load strings for different languages easily. The translations can be stored in a seperate toml file and is applied with the  `linguify()` function. There are currently different workflows depending on package or template usage. 

### License

MIT

**Comment by huwaireb** [comment]
Hello,

I've sent an invite to the organization, once you accept it you can transfer the repository over.

[Transferring a Repository Owned by your Personal Account](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account)
